### PR TITLE
 Test jruby on evergreen 

### PR DESCRIPTION
**Pull request by p-mongo**
Depends on https://github.com/10gen/mongo-ruby-toolchain/pull/2

https://jira.mongodb.org/browse/RUBY-1377

### CODE REVIEW

**Comment by p-mongo** [comment]
My rbenv efforts failed due to it being in different places when building the toolchain vs running tests. Will try chruby next.

**Comment by p-mongo** [comment]
Currently blocked on https://github.com/10gen/buildhost-configuration/pull/107

File: spec/support/transactions.rb [hunk 1]
```diff
@@ -65,7 +65,8 @@ class Spec
       # @since 2.6.0
       def initialize(file)
         file = File.new(file)
-        @spec = YAML.load(ERB.new(file.read).result)
+        contents = ERB.new(file.read).result
```
Comment (saghm) [inline spec/support/transactions.rb]: Are all of the changes in this file necessary for enabling JRuby testing? It doesn't look like any of them are related

**Comment by saghm** [comment]
Not sure if this is still necessary or if it's superseded by #1008

File: spec/support/transactions.rb (hunk 1 above)
Comment (p-mongo) [reply spec/support/transactions.rb]: Yes they are all necessary.

**Comment by p-mongo** [comment]
I am sure: this is still necessary.

**Pull request merged**